cmd/kubeadm/app/api/types.go

```diff
@@ -71,6 +71,8 @@ const (
 	DefaultServiceDNSDomain   = "cluster.local"
 	DefaultServicesCIDRString = "100.64.0.0/12" // Carrier-grade NAT range (RFC 6598)
 	DefaultKubernetesVersion  = "v1.4.0-beta.10"
+	DefaultCloudConfigPath    = "/etc/kubernetes/cloud-config.json"
```

Comment:
It makes more sense there plus being able to output it for the user if there's need to